template/assets/scss/spacing.scss

```diff
@@ -51,5 +51,39 @@ $positions: (
         .#{$paddingKey}#{$posKey}#{$separator}#{$sizeKey} {
             padding-#{$posValue}: sizeValue($sizeKey, $sizeValue) !important;
         }
+
+        // these util classes for different screen sizes are dependent on bulma's mixin for screen sizes
+        @include mobile {
+            .#{$paddingKey}#{$posKey}#{$separator}#{$sizeKey}-is-mobile {
```

Comment:
I think my preference would be to drop the `-is` and just leave them as `-mobile` or `-mobile-only`.  It's a little closer in line with how Bulma handles size modifiers in class names.

e.g.,  `.pb-xs-mobile` or `.pb-xs-mobile-only`

Also, should we add 'widescreen' (1200+px) while we're at it?